root/search/components/RecordingResults.js

```diff
@@ -124,7 +124,7 @@ export const RecordingResultsInline = ({
         <th>{l('Artist')}</th>
         <th>{l('ISRCs')}</th>
         <th>{l('Release')}</th>
-        {$c.session && $c.session.tport ? <th>{l('Tagger')}</th> : null}
+        {$c && $c.session && $c.session.tport ? <th>{l('Tagger')}</th> : null}
```

Comment:
This can possibly also be substituted by something else if desired, but appears only once.